Bigtable: failing system test

/cc @sduskis, @vikas-jamdar (bug introduced in PR #5790).

From:

- https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/7763
- https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/7766

```python
___________ TestInstanceAdminAPI.test_create_instance_w_two_clusters ___________

self = <tests.system.TestInstanceAdminAPI testMethod=test_create_instance_w_two_clusters>

    def test_create_instance_w_two_clusters(self):
        from google.cloud.bigtable import enums
        _PRODUCTION = enums.Instance.Type.PRODUCTION
        ALT_INSTANCE_ID = 'dif' + unique_resource_id('-')
        instance = Config.CLIENT.instance(ALT_INSTANCE_ID,
                                          instance_type=_PRODUCTION,
                                          labels=LABELS)
    
        ALT_CLUSTER_ID_1 = ALT_INSTANCE_ID+'-c1'
        ALT_CLUSTER_ID_2 = ALT_INSTANCE_ID+'-c2'
        LOCATION_ID_2 = 'us-central1-f'
        STORAGE_TYPE = enums.StorageType.HDD
        cluster_1 = instance.cluster(
            ALT_CLUSTER_ID_1, location_id=LOCATION_ID, serve_nodes=SERVE_NODES,
            default_storage_type=STORAGE_TYPE)
        cluster_2 = instance.cluster(
            ALT_CLUSTER_ID_2, location_id=LOCATION_ID_2,
            serve_nodes=SERVE_NODES, default_storage_type=STORAGE_TYPE)
        operation = instance.create(clusters=[cluster_1, cluster_2])
        # We want to make sure the operation completes.
        operation.result(timeout=10)
    
        # Make sure this instance gets deleted after the test case.
        self.instances_to_delete.append(instance)
    
        # Create a new instance instance and make sure it is the same.
        instance_alt = Config.CLIENT.instance(ALT_INSTANCE_ID)
        instance_alt.reload()
    
        self.assertEqual(instance, instance_alt)
        self.assertEqual(instance.display_name, instance_alt.display_name)
        self.assertEqual(instance.type_, instance_alt.type_)
    
        clusters, failed_locations = instance_alt.list_clusters()
        self.assertEqual(failed_locations, [])
    
        clusters.sort(key=lambda x: x.name)
        alt_cluster_1, alt_cluster_2 = clusters
    
        self.assertEqual(cluster_1.location_id, alt_cluster_1.location_id)
        self.assertEqual(alt_cluster_1.state, enums.Cluster.State.READY)
        self.assertEqual(cluster_1.serve_nodes, alt_cluster_1.serve_nodes)
        self.assertEqual(cluster_1.default_storage_type,
                         alt_cluster_1.default_storage_type)
        self.assertEqual(cluster_2.location_id, alt_cluster_2.location_id)
        self.assertEqual(alt_cluster_2.state, enums.Cluster.State.READY)
        self.assertEqual(cluster_2.serve_nodes, alt_cluster_2.serve_nodes)
        self.assertEqual(cluster_2.default_storage_type,
                         alt_cluster_2.default_storage_type)
    
        # Test list clusters in project via 'client.list_clusters'
        clusters, failed_locations = Config.CLIENT.list_clusters()
        self.assertFalse(failed_locations)
        found = set([cluster.name for cluster in clusters])
        self.assertTrue({alt_cluster_1.name,
                         alt_cluster_2.name,
                         Config.CLUSTER.name}.issubset(found))
        self._test_state_helper(instance, ALT_CLUSTER_ID_1,
>                               ALT_CLUSTER_ID_2)

tests/system.py:277: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tests.system.TestInstanceAdminAPI testMethod=test_create_instance_w_two_clusters>
instance = <google.cloud.bigtable.instance.Instance object at 0x7f2c4009e350>
clusterid1 = 'dif-7763-1534881854-c1', clusterid2 = 'dif-7763-1534881854-c2'

    def _test_state_helper(self, instance, clusterid1, clusterid2):
        # test get_cluster_states for a table in instance
        from google.cloud.bigtable.enums import Table as enum_table
        from google.cloud.bigtable.table import ClusterState
        STATE_NOT_KNOWN = enum_table.ReplicationState.STATE_NOT_KNOWN
        INITIALIZING = enum_table.ReplicationState.INITIALIZING
        PLANNED_MAINTENANCE = enum_table.ReplicationState. \
            PLANNED_MAINTENANCE
        UNPLANNED_MAINTENANCE = enum_table.ReplicationState. \
            UNPLANNED_MAINTENANCE
        READY = enum_table.ReplicationState.READY
        temp_table_id = 'test-get-cluster-states'
        temp_table = instance.table(temp_table_id)
        temp_table.create()
        result = temp_table.get_cluster_states()
        expected_results = [
            ClusterState(STATE_NOT_KNOWN),
            ClusterState(INITIALIZING),
            ClusterState(PLANNED_MAINTENANCE),
            ClusterState(UNPLANNED_MAINTENANCE),
            ClusterState(READY)
        ]
        cluster_id_list = result.keys()
        self.assertEqual(len(cluster_id_list), 2)
        self.assertIn(clusterid1, cluster_id_list)
        self.assertIn(clusterid2, cluster_id_list)
        for clusterstate in result.values():
            self.assertIn(clusterstate, expected_results)
    
        # Test create app profile with multi_cluster_routing policy
        app_profiles_to_delete = []
        description = 'routing policy-multy'
        app_profile_id_1 = 'app_profile_id_1'
>       routing = enums.RoutingPolicyType.ANY
E       NameError: global name 'enums' is not defined
```

--- a/bigtable/tests/system.py
+++ b/bigtable/tests/system.py
@@ -218,14 +218,15 @@ class TestInstanceAdminAPI(unittest.TestCase):
 
     def test_create_instance_w_two_clusters(self):
         from google.cloud.bigtable import enums
+        from google.cloud.bigtable.table import ClusterState
         _PRODUCTION = enums.Instance.Type.PRODUCTION
         ALT_INSTANCE_ID = 'dif' + unique_resource_id('-')
         instance = Config.CLIENT.instance(ALT_INSTANCE_ID,
                                           instance_type=_PRODUCTION,
                                           labels=LABELS)
 
-        ALT_CLUSTER_ID_1 = ALT_INSTANCE_ID+'-c1'
-        ALT_CLUSTER_ID_2 = ALT_INSTANCE_ID+'-c2'
+        ALT_CLUSTER_ID_1 = ALT_INSTANCE_ID + '-c1'
+        ALT_CLUSTER_ID_2 = ALT_INSTANCE_ID + '-c2'
         LOCATION_ID_2 = 'us-central1-f'
         STORAGE_TYPE = enums.StorageType.HDD
         cluster_1 = instance.cluster(
@@ -273,35 +274,23 @@ class TestInstanceAdminAPI(unittest.TestCase):
         self.assertTrue({alt_cluster_1.name,
                          alt_cluster_2.name,
                          Config.CLUSTER.name}.issubset(found))
-        self._test_state_helper(instance, ALT_CLUSTER_ID_1,
-                                ALT_CLUSTER_ID_2)
 
-    def _test_state_helper(self, instance, clusterid1, clusterid2):
-        # test get_cluster_states for a table in instance
-        from google.cloud.bigtable.enums import Table as enum_table
-        from google.cloud.bigtable.table import ClusterState
-        STATE_NOT_KNOWN = enum_table.ReplicationState.STATE_NOT_KNOWN
-        INITIALIZING = enum_table.ReplicationState.INITIALIZING
-        PLANNED_MAINTENANCE = enum_table.ReplicationState. \
-            PLANNED_MAINTENANCE
-        UNPLANNED_MAINTENANCE = enum_table.ReplicationState. \
-            UNPLANNED_MAINTENANCE
-        READY = enum_table.ReplicationState.READY
         temp_table_id = 'test-get-cluster-states'
         temp_table = instance.table(temp_table_id)
         temp_table.create()
         result = temp_table.get_cluster_states()
+        ReplicationState = enums.Table.ReplicationState
         expected_results = [
-            ClusterState(STATE_NOT_KNOWN),
-            ClusterState(INITIALIZING),
-            ClusterState(PLANNED_MAINTENANCE),
-            ClusterState(UNPLANNED_MAINTENANCE),
-            ClusterState(READY)
+            ClusterState(ReplicationState.STATE_NOT_KNOWN),
+            ClusterState(ReplicationState.INITIALIZING),
+            ClusterState(ReplicationState.PLANNED_MAINTENANCE),
+            ClusterState(ReplicationState.UNPLANNED_MAINTENANCE),
+            ClusterState(ReplicationState.READY)
         ]
         cluster_id_list = result.keys()
         self.assertEqual(len(cluster_id_list), 2)
-        self.assertIn(clusterid1, cluster_id_list)
-        self.assertIn(clusterid2, cluster_id_list)
+        self.assertIn(ALT_CLUSTER_ID_1, cluster_id_list)
+        self.assertIn(ALT_CLUSTER_ID_2, cluster_id_list)
         for clusterstate in result.values():
             self.assertIn(clusterstate, expected_results)
 

